addons/l10n_ar_withholding/views/product_template_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="name">product.template.product.form</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="account.product_template_form_view"/>
+        <field name="arch" type="xml">
+            <field name="supplier_taxes_id" position="after">
+                <field name="l10n_ar_supplier_withholding_taxes_ids" widget="many2many_tags"  invisible="'AR'.lower() not in (fiscal_country_codes or '').lower()" options="{'no_create': True}"/>
```

Comment:
@jco-odoo I did this so that it would look the same as in other places. e.g.  
https://github.com/odoo/enterprise/blob/master/l10n_co_edi/views/product_template_views.xml#L10 
I keep the current syntax or I put the cleaner version of the expression

Reply:
A branch is being merged replacing it elsewhere, so change it

Reply:
done